persistence.js

```diff
@@ -90,7 +90,7 @@ async function * createWillStream (db, brokers, maxWills) {
   const results = await db.lrange(WILLSKEY, 0, maxWills)
   for (const key of results) {
     const result = await getDecodedValue(db, WILLSKEY, key)
-    if (!brokers || !brokers[result.split(':')[1]]) {
+    if (!brokers || !brokers[result.brokerId.split(':')[0]]) {
```

Comment:
why are you using split with brokerId? Split would only be needed if you use `key`. 

```suggestion
    if (!brokers || !brokers[key.split(':')[1]]) {
```

Reply:
If you check here: https://github.com/moscajs/aedes-persistence-redis/commit/b8b534a504b1d998dac6bd950f1abf95e7f95cd2#diff-58943acef9db8f8aea955bd46f71e484c4f318473dfdf1fb5f0a4aaecf789a9dL591


`result` was your actual `key`

Reply:
Right, thanks for clarifying that!  I'll fix it tonight.